dotnet-mfa-example/Views/Home/ChallengeSuccess.cshtml

```diff
@@ -0,0 +1,44 @@
+@{
+    ViewData["Title"] = "Challenge Success";
+}
+
+<body class="container_success">
+    <div class="logged_in_nav">
+        <div class="flex">
+            <p>MFA Example Application</p>
+        </div>
+        <div>
+            <img src="../static/images/workos_logo_new.png" alt="workos logo">
+        </div>
+    </div>
+    <div class='flex'>
+        <div class="logged_in_div_left">
+            <div>
+                <h1>Your app,</h1>
+                <h2>Enterprise Ready</h2>
+            </div>
+            <div>
+                <a href="https://workos.com/" target="_blank"><button class='button'>WorkOS</button></a>
+                <a href="https://workos.com/docs" target="_blank"><button class='button'>Documentation</button></a>
+                <a href="https://workos.com/docs/reference" target="_blank"><button class='button'>API
+                        Reference</button></a>
+                <a href="https://workos.com/blog" target="_blank"><button class='button'>Blog</button></a>
+
+            </div>
+        </div>
+        <div class="logged_in_div_right">
+            <div class="flex_column">
+                <div class="flex-column">
+                    <div class="card">
+                        <h1>User Verified: @ViewBag.successFactor.Valid</h1>
+                        <p>Factor ID: @ViewBag.successFactor.Challenge.FactorId</p>
+                        <p>Created At: @ViewBag.successFactor.Challenge.CreatedAt</p>
+                        <p>Expires At: @ViewBag.successFactor.Challenge.ExpiresAt</p>
```

Comment:
The Expires at field only applies to 'sms' type factors. Can that be conditionally rendered here so it doesn't appear for 'totp'?